test/unit/rtc_session.js

```diff
@@ -77,10 +77,12 @@ describe('RTC session', () => {
 
         it('skips gUM call if user media stream is already provided', () => {
             session._state = state;
-            session._userAudioStream = {};
+            session._userProvidedStream = true;
             session.transit = sinon.spy();
+            session._gUM = sinon.spy();
             state.onEnter();
             chai.assert(session.transit.calledOnce);
+            chai.assert(session._gUM.notCalled);
             chai.assert(session.transit.args[0][0] instanceof CreateOfferState);
```

Comment:
Same as above, can we assert absolute values in addition to checking instance type?

Reply:
These are state transitions that are asserted. The absolute value is a dynamically allocated object. Not sure if we can assert on absolute value here